### DISCUSSION TITLE
Can not open channels

### DISCUSSION THREAD

**Discussion by Nikefreak** [Ride-The-Lightning/RTL#1324] [Troubleshooting Questions] [answered]
<img width="832" alt="Bildschirmfoto 2023-11-09 um 16 30 16" src="https://github.com/Ride-The-Lightning/RTL/assets/113097053/0eb4cd39-6044-46b0-bbf9-1c290f27f276">

Can somebody tell my why i can not open any channels?

Thanks!

**Comment by saubyk** [comment, marked as answer]
Hi @Nikefreak it appears that you don't have enough funds to pay the fee rate you're trying to set for this channel opening transaction. You have a total funds of 65391 sats and you're trying to open a channel of 50000 sats, I don't think there will be enough funds left in your wallet to pay a fee rate of 100 sats/vb

My recommendation is to add additional funds to your node and attempt to open channel with larger amount. Please note that it's not recommended to open channels with such a low amount as 50000 sats. Given the current fee env in the mempool, you'll pay a lot trying to open and close such low value channels and not able to use the LN much.

**Reply by Nikefreak** [reply]
Hi @saubyk, thanks for your answer! Thats just a test flight, i just want to open a channel to do some test transfers. After that i want to open bigger channels.

For your info: I have 115,391 sats in total (50000 + 65391 sats). This should be enough value left to open a channel with 50k sats. That makes no sense to me..

**Reply by saubyk** [reply]
ah, I miss read the values. Have you tried opening with another peer?
Also curious to know, how many UTXOs you have on the node? And their respective values?

You should be able to find that on the onchain page

**Reply by Nikefreak** [reply]
![Unbenat](https://github.com/Ride-The-Lightning/RTL/assets/113097053/6b403789-c660-44b2-8847-b8c79c77e4d9)

You can find my UTXO overview here @saubyk

**Reply by Nikefreak** [reply]
It is working now! I connected my node with the 1ML Alpha Node without any advanced settings at the channel opening settings. Before i restarted my node.

Thanks for your help/time! @saubyk
